src/datachain/lib/dc.py

```diff
@@ -290,18 +276,43 @@ def column(self, name: str) -> Column:
     def c(self, column: Union[str, Column]) -> Column:
         """Returns Column instance attached to the current chain."""
         c = self.column(column) if isinstance(column, str) else self.column(column.name)
-        c.table = self.table
+        c.table = self._query.table
         return c
 
+    @property
+    def session(self) -> Session:
+        """Session of the chain."""
+        return self._query.session
+
+    @property
+    def name(self) -> Optional[str]:
+        """Name of the chain."""
```

Comment:
what is `name` of the chain? and version? are those dataset name / version or something else?

Reply:
Yes, it's the name/version of the dataset.

Reply:
let's please rephrase it then? wdyt?